### PR TITLE
Adds support for overriding the new cf color variables coming to cf-buttons.

### PR DESCRIPTION
**Pull request by himedlooff**

### CODE REVIEW

File: app/templates/src/static/css/main.less [hunk 1]
```diff
@@ -1,14 +1,26 @@
...
+
+// Override the Capital Framework theme variables with colors from the CFPB color palette.
+@import (less) "cf-theme-overrides.less";
+
+
+// ==========================================================================
+// This is your project's main Less file. Project-specific Less rules go below.
 // We've generated some sample rules for you. They're used with the index.html file.
+// Please delete them before starting your project.
+// ==========================================================================
```
Comment (Scotchester) [inline app/templates/src/static/css/main.less]: I think this comment block should go above the two nearest imports above, so third parties don't accidentally skip past them.

Comment (himedlooff) [inline app/templates/src/static/css/main.less]: Updated, is that better?

Comment (Scotchester) [inline app/templates/src/static/css/main.less]: Yup, that works, too.

**Comment by Scotchester** [comment]
:+1:

**Comment by himedlooff** [comment]
Thanks for the review! :smile:

**Pull request merged**